scripts/AddIdentity.C

```diff
@@ -0,0 +1,18 @@
+/**
```

Comment:
Remove this file, such an identifier already exists in the event header.

Reply:
This identifier is used to identify separate reconstruction techniques. Is that in the event header?

Reply:
you could easily do what you want with the existing header, yes

Reply:
You just have to fill it correctly